### PR TITLE
AC Automated Test - LRAC-9052 | master

### PR DESCRIPTION
**Pull request by rafaellajordao**
https://issues.liferay.com/browse/LRAC-9052

### CODE REVIEW

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by christianstokes** [comment]
@rafaellajordao looks great, even though it is a short test, can you add some "task" descriptions? Thank you!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase [hunk 1]
```diff
@@ -40,12 +40,19 @@ definition {
...
+		ACUtils.launchAC();
+
+		ACNavigation.goToSettings();
+
+		ACSettings.goToUserManagement();
+
+		ACUtils.searchBar(searchItem = "corbin.murakami@faro.io");
 
-		// TODO LRAC-9052 AdminCanBeSelected
+		Check.checkToggleSwitch(locator1 = "ACSettings#USER_LIST_CHECKBOX");
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase:53]: Hi @rafaellajordao, I think this `ACInterests.clickKeywordCheckbox` macro has the same function as your command. I think that you can put this macro in a file that is more generic

Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase]: fixed!! @ViniLopes20

**Comment by rafaellajordao** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase [hunk 2]
```diff
@@ -28,12 +28,21 @@ definition {
...
 	@priority = "3"
 	test AdminCanBeSelected {
+		task ("Go to settings and go to user management") {
+			ACNavigation.goToSettings();
+
+			ACSettings.goToUserManagement();
+		}
 
-		// TODO LRAC-9052 AdminCanBeSelected
+		task ("Search for a adm user and check if it is possible to select") {
```
Comment (christianstokes) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase:39]: typo, "adm" :)

Comment (rafaellajordao) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ManageUsers.testcase]: fixed!! @christianstokes

**Comment by rafaellajordao** [comment]
ci:test:sf

**Comment by alexcandidos** [comment]
Hi @rafaellajordao , Thanks for working on it!

The code looks good to me!

I'm forwarding it.

**Comment by alexcandidos** [comment]
ci:forward

**Pull request closed without merging**